### PR TITLE
GDX samples: remember map position

### PR DESCRIPTION
**Pull request by Gustl22**
For easier debugging...

### CODE REVIEW

File: vtm-android-example/src/org/oscim/android/test/GdxActivity.java [hunk 1]
```diff
@@ -65,6 +70,17 @@ public void createLayers() {
                     .httpFactory(new OkHttpEngine.OkHttpFactory())
                     .build();
             initDefaultLayers(ts, false, true, true);
+            resume();
```
Comment (devemux86) [inline vtm-android-example/src/org/oscim/android/test/GdxActivity.java]: Better let life cycle calls to the system.

File: vtm-android-example/src/org/oscim/android/test/GdxActivity.java [hunk 2]
```diff
@@ -65,6 +70,17 @@ public void createLayers() {
...
+        }
+
+        @Override
+        public void pause() {
+            mPrefs.save(getMap());
+        }
```
Comment (devemux86) [inline vtm-android-example/src/org/oscim/android/test/GdxActivity.java]: For clarity could include all the `super` calls.

File: vtm-android-example/src/org/oscim/android/test/GdxActivity.java [hunk 3]
```diff
@@ -65,6 +70,17 @@ public void createLayers() {
...
+
+        @Override
+        public void resume() {
+            mPrefs.load(getMap());
         }
```
Comment (devemux86) [inline vtm-android-example/src/org/oscim/android/test/GdxActivity.java]: `resume` is called here in GDX life cycle, can move the prefs load above.

**Comment by devemux86** [comment]
Thanks!
Merged via f3f629779f9cd387c6c9a8433535dd639a872791 with the aforementioned changes.

File: vtm-android-example/src/org/oscim/android/test/GdxActivity.java (hunk 1 above)
Comment (Gustl22) [reply vtm-android-example/src/org/oscim/android/test/GdxActivity.java]: resume wasn't called by system, kind of strange... :see_no_evil:

**Pull request closed without merging**